mangaki/mangaki/utils/mal.py

```diff
@@ -17,17 +17,29 @@
 from django.contrib.auth.models import User
 from django.contrib.postgres.search import SearchQuery
 from django.db.models import QuerySet, Q
+from django.utils.functional import cached_property
 from django.db import transaction
 
-from mangaki.models import Work, Rating, SearchIssue, Category, WorkTitle
-
-from collections import namedtuple
+from mangaki.models import Work, Rating, SearchIssue, Category, WorkTitle, ExtLanguage
 
 import logging
 
 logger = logging.getLogger(__name__)
 
 
+# refer to 0072 migration to understand why ext_lang is set to these values.
```

Comment:
This should be the inverse: explain here, have a reference to this code in 0072.